components/Molecule/navigation-component.html

```diff
@@ -0,0 +1,72 @@
+<div class="component"  title="Navigation Component" version="1">
+
+    <style>
+.navigation-components{
+    margin: 100px 100px;
+}
+
+#main-nav li{
+  text-align: center;
+}
+aside{
+    display: none;
+    background-color: whitesmoke;
+    float: right;
+    width: 25.5%;
+    border-left: 2px solid #9aa4a5;
+    border-bottom: 2px solid #9aa4a5;
+    border-top: 2px solid #9aa4a5;
+
+}
+ nav{
+     width: 100%;
+ }
+li{
+    list-style: none;
+    line-height: 2rem;
+    font-size: 1em;
+
+}
+li a {
+    text-decoration: none;
+    color: #444;
+
+}
+
+  </style>
+
+  <markup>
+      <div class="container">
+          <nav id="main-nav">
+            <ul>
+              <li><a href="#main">Page Names</a></li>
+            </ul>
+          </nav>
+
+     <div class="navigation-components">
+
```

Comment:
You probably don't need these lines (39-46) as you are creating just one molecule.

Reply:
@MustafaAlsaeedi Thank you so much for your feedback, that is very helpful and knowledgeable. You guys teach us what does mean code review. On the other hand, I added line 39-46 to show there will be the main banner. And if you see the Casa Marianella layout the aside navigation component display when you click the **About** from the main banner menu. So, I used the property and value **display: none** and then I added the click add an event listener using JavaScript.